src/etc/inc/interfaces.inc

```diff
@@ -3039,33 +3039,61 @@ function find_dhcp6c_process($interface) {
 	return intval($pid);
 }
 
-function kill_dhcp6client_process($interface, $norelease) {
+function kill_dhcp6client_process($interface,$Force,$release = false) {
```

Comment:
`$Force` should be lowercase (`$force`) with all instances changed to follow.

There should be spaces between the parameters in the list.